net/tlsdial/tlsdial.go

```diff
@@ -44,6 +45,16 @@ var debug = envknob.RegisterBool("TS_DEBUG_TLS_DIAL")
 // Headscale, etc.
 var tlsdialWarningPrinted sync.Map // map[string]bool
 
+var mitmBlockWarnable = *health.Register(&health.Warnable{
```

Comment:
drop the `*` deference here. We don't do that elsewhere.

Reply:
done